src/tools/genpolicy/rules.rego

```diff
@@ -56,13 +54,198 @@ S_NAME_KEY = "io.kubernetes.cri.sandbox-name"
 S_NAMESPACE_KEY = "io.kubernetes.cri.sandbox-namespace"
 BUNDLE_ID = "[a-z0-9]{64}"
 
-CreateContainerRequest:= {"ops": ops, "allowed": true} {
+CreateContainerRequest:= resp {
+    not is_null(input.env_map)
+    
+    i_env_map := input.env_map
+    
+    some p_container in policy_data.containers
+    p_env_map := p_container.env_map  
+    allow_env_map(p_env_map, i_env_map)
+    
+    i_process := input.base.OCI.Process
+    
+    p_process := p_container.OCI.Process
+    
+    allow_args(i_process, p_process, i_env_map)
+
+    resp = CreateContainerRequestCommon(input.base)
+
+    print("PolicyCreateContainerRequest: true")
```

Comment:
am I reading wrong or is this misplaced?

Reply:
You are right. This should be CreateContainerRequest, and the other CreateContainerRequest logs below should be updated to CreateContainerRequest2